pyown/items/camera/camera.py

```diff
@@ -0,0 +1,273 @@
+        await self._send_command_without_where(WhatCamera.DECREASE_Y)
+
+    async def increase_luminosity(self):
+        """Increases the luminosity of the camera image."""
+        await self._send_command_without_where(WhatCamera.INCREASE_LUMINOSITY)
+
+    async def decrease_luminosity(self):
+        """Decreases the luminosity of the camera image."""
+        await self._send_command_without_where(WhatCamera.DECREASE_LUMINOSITY)
+
+    async def increase_contrast(self):
+        """Increases the contrast of the camera image."""
+        await self._send_command_without_where(WhatCamera.INCREASE_CONTRAST)
+
+    async def decrease_contrast(self):
+        """Decreases the contrast of the camera image."""
+        await self._send_command_without_where(WhatCamera.DECREASE_CONTRAST)
+
+    async def increase_color(self):
+        """Increases the color saturation of the camera image."""
+        await self._send_command_without_where(WhatCamera.INCREASE_COLOR)
+
+    async def decrease_color(self):
+        """Decreases the color saturation of the camera image."""
+        await self._send_command_without_where(WhatCamera.DECREASE_COLOR)
+
+    async def increase_quality(self):
+        """Increases the quality of the camera image."""
+        await self._send_command_without_where(WhatCamera.INCREASE_QUALITY)
+
+    async def decrease_quality(self):
+        """Decreases the quality of the camera image."""
+        await self._send_command_without_where(WhatCamera.DECREASE_QUALITY)
+
+    async def display_dial(self, x: int, y: int):
+        """
+        Displays a specific dial position.
+        
+        Args:
+            x: The X dial number (1-4).
+            y: The Y dial number (1-4).
+            
+        Raises:
+            ValueError: If x or y are not in the range 1-4.
+        """
+        if x not in range(1, 5) or y not in range(1, 5):
+            raise ValueError("Dial coordinates must be in range 1-4")
+        
+        dial_map = {
+            (1, 1): WhatCamera.DISPLAY_DIAL_11,
+            (1, 2): WhatCamera.DISPLAY_DIAL_12,
+            (1, 3): WhatCamera.DISPLAY_DIAL_13,
+            (1, 4): WhatCamera.DISPLAY_DIAL_14,
+            (2, 1): WhatCamera.DISPLAY_DIAL_21,
+            (2, 2): WhatCamera.DISPLAY_DIAL_22,
+            (2, 3): WhatCamera.DISPLAY_DIAL_23,
+            (2, 4): WhatCamera.DISPLAY_DIAL_24,
+            (3, 1): WhatCamera.DISPLAY_DIAL_31,
+            (3, 2): WhatCamera.DISPLAY_DIAL_32,
+            (3, 3): WhatCamera.DISPLAY_DIAL_33,
+            (3, 4): WhatCamera.DISPLAY_DIAL_34,
+            (4, 1): WhatCamera.DISPLAY_DIAL_41,
+            (4, 2): WhatCamera.DISPLAY_DIAL_42,
+            (4, 3): WhatCamera.DISPLAY_DIAL_43,
+            (4, 4): WhatCamera.DISPLAY_DIAL_44,
+        }
+        
+        await self._send_command_without_where(dial_map[(x, y)])
+
+    @classmethod
+    def on_status_change(
+        cls, callback: Callable[[Self, WhatCamera], Coroutine[None, None, None]]
```

Comment:
The callback typing is wrong. The callable should follow the types of the arguments passed on call_callbacks, also the call_callbacks should also pass as an argument the message received